lib/active_uuid/railtie.rb

```diff
@@ -1,12 +1,13 @@
 require "active_uuid"
+require "active_uuid/connection_patches"
 require "rails"
 
 module ActiveUUID
   class Railtie < Rails::Railtie
     railtie_name :activeuuid
 
     config.to_prepare do
-      ActiveUUID::Patches.apply!
+      ActiveUUID::ConnectionPatches.apply!
```

Comment:
This very invocation concludes `active_uuid/connection_patches.rb`, so I suppose moving line no. 2 here is a better option.

Reply:
By line no. 2, is that `require "rails` or `ActiveUUID::ConnectionPatches.apply!` or some other lines?

Where does the line need to be moved?

Reply:
I have amended that commit. See: c4ffad273defa9092bed967f587f5bd066271ae1.

Reply:
Nevertheless, I'm not sure if we need Railtie at all. I don't like that it applies patches unconditionally. See my other comment: https://github.com/riboseinc/activeuuid/pull/55#issuecomment-472820014.